app/models/giveaway.rb

```diff
@@ -13,12 +12,16 @@ def choose_winners
                 return 
             else
                 w.item_won = product[:prod_id]
-                winners << w
+                w.save
             end
         end
       end
     end
 
+    def winners 
+        users.where("item_won <> ''")
```

Comment:
Yeah I figured that haha, I mean moving the `where("item_won <> ''")` into `user.rb`.
Again only benefit is if you think you'll use it elsewhere.